contrib/buildsystems/CMakeLists.txt

```diff
@@ -35,25 +40,38 @@ empty(default) :
 NOTE: -DCMAKE_BUILD_TYPE is optional. For multi-config generators like Visual Studio
 this option is ignored
 
-This process generates a Makefile(Linux/*BSD/MacOS) , Visual Studio solution(Windows) by default.
+This process generates: a Makefile(Linux/*BSD/MacOS), Visual Studio solution(Windows) by default.
 Run `make` to build Git on Linux/*BSD/MacOS.
 Open git.sln on Windows and build Git.
 
 NOTE: By default CMake uses Makefile as the build tool on Linux and Visual Studio in Windows,
 to use another tool say `ninja` add this to the command line when configuring.
 `-G Ninja`
 
+The Visual Studio default generator changed in v16.6 from its visual studio
+implemenation to `Ninja` This required changes to many CMake scripts.
+
 ]]
 cmake_minimum_required(VERSION 3.14)
 
 #set the source directory to root of git
 set(CMAKE_SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/../..)
 if(WIN32)
 	set(VCPKG_DIR "${CMAKE_SOURCE_DIR}/compat/vcbuild/vcpkg")
+	message("WIN32: ${WIN32}")
```

Comment:
Why print out `WIN32` when it is a Boolean and we're inside the conditional block where we _know_ that it is `true`?

Reply:
> Why print out `WIN32` when it is a Boolean and we're inside the conditional block where we _know_ that it is `true`?

Thanks for the prompt comments.

While we may assume it's evaluated to `true`, it's not obvious what it's actual text value is (or has become in future cases). 

I found that much of the CMake documentation left a lot to be desired about those sorts of things (confusions between existence, particular text values etc. Previously it was part of a logic term (`AND MSVC` iirc).